tests/test_webviz_subsurface_testdata.py

```diff
@@ -0,0 +1,119 @@
+"""Testing loading the webviz subsurface testdata."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import datetime
```

Comment:
again, consider the simpler `time` package? (not sure if there is any efficiency gain though, but at least less verbose code)

Reply:
removd.